pink/src/pb_cli.cc

```diff
@@ -51,7 +51,7 @@ Status PbCli::Send(void *msg) {
   google::protobuf::Message *req =
     reinterpret_cast<google::protobuf::Message *>(msg);
 
-  int wbuf_len = req->ByteSize();
+  int wbuf_len = req->ByteSizeLong();
```

Comment:
Why?

Reply:
Fixing a compiler warning, it used a deprecated function.